pkg/certproviderutil/util.go

```diff
@@ -9,161 +9,175 @@ import (
 	cmversionedclient "github.com/jetstack/cert-manager/pkg/client/clientset/versioned"
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
 
 	"github.com/openservicemesh/osm/pkg/certificate"
 	"github.com/openservicemesh/osm/pkg/certificate/providers/certmanager"
 	"github.com/openservicemesh/osm/pkg/certificate/providers/tresor"
 	"github.com/openservicemesh/osm/pkg/certificate/providers/vault"
-	"github.com/openservicemesh/osm/pkg/configurator"
 	"github.com/openservicemesh/osm/pkg/constants"
 	"github.com/openservicemesh/osm/pkg/debugger"
 )
 
 // These are the supported certificate issuers.
 const (
 	// Tresor is an internal package, which leverages Kubernetes secrets and signs certs on the OSM pod
-	tresorKind string = "tresor"
+	TresorKind string = "tresor"
 
 	// Hashi Vault integration; OSM is pointed to an external Vault; signing of certs happens on Vault
-	vaultKind = "vault"
+	VaultKind = "vault"
 
 	// cert-manager integration; certificates are requested using cert-manager
 	// CertificateRequest resources, signed by the configured issuer.
-	certmanagerKind = "cert-manager"
+	CertManagerKind = "cert-manager"
 
 	// Additional values for the root certificate
 	rootCertCountry      = "US"
 	rootCertLocality     = "CA"
 	rootCertOrganization = "Open Service Mesh"
```

Comment:
I took the liberty to create a GitHub Issue to make these configurable:  https://github.com/openservicemesh/osm/issues/2470